thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/alert/DetectionAlertFilterNotification.java

```diff
@@ -27,19 +28,33 @@
  * Container class for notification properties
  */
 public class DetectionAlertFilterNotification {
+
   Map<String, Object> notificationSchemeProps;
+  Map<String, String> dimensionFilters;
 
   public DetectionAlertFilterNotification(Map<String, Object> notificationSchemeProps) {
+    this(notificationSchemeProps, new HashMap<>());
+  }
+
+  public DetectionAlertFilterNotification(Map<String, Object> notificationSchemeProps, Map<String, String> dimensionFilters) {
     this.notificationSchemeProps = notificationSchemeProps;
+    this.dimensionFilters = dimensionFilters;
   }
 
   public Map<String, Object> getNotificationSchemeProps() {
     return notificationSchemeProps;
   }
 
-  public DetectionAlertFilterNotification setNotificationSchemeProps(Map<String, Object> notificationSchemeProps) {
+  public void setNotificationSchemeProps(Map<String, Object> notificationSchemeProps) {
     this.notificationSchemeProps = notificationSchemeProps;
-    return this;
+  }
+
+  public Map<String, String> getDimensionFilters() {
```

Comment:
Is this `dimensionFilters`  the same with the `DimensionMap` in the anomaly? shall we use MultiMap instead of Map?

Reply:
it is not necessary but probably better to support it. I will update the changes.
Thanks!